web/modules/custom/vote_utility/inc/token.inc

```diff
@@ -54,20 +54,23 @@ function vote_utility_tokens($type, $tokens, array $data, array $options, Bubble
 }
 
 /**
- * Get token replacement value, based on value of menu location on an entity otherwise fall back on node title.
+ * Gets token replacement value, based on value of menu location on an entity.
```

Comment:
I broke this up into 2 comments after look though the docs I was not able to get the steps to pass after working though it and was able to get this to flow as well